Content.Server/Actions/Spells/GiveItemSpell.cs

```diff
@@ -0,0 +1,56 @@
+using Content.Server.Hands.Components;
+using Content.Server.Items;
+using Content.Server.Notification;
+using Content.Shared.ActionBlocker;
+using Content.Shared.Actions.Behaviors;
+using Content.Shared.Actions.Components;
+using Content.Shared.Cooldown;
+using Content.Shared.Notification.Managers;
+using JetBrains.Annotations;
+using Robust.Shared.Audio;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Robust.Shared.Player;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Server.Actions.Spells
+{
+    [UsedImplicitly]
+    [DataDefinition]
+    public class GiveItemSpell : IInstantAction
+    {
+        [ViewVariables] [DataField("castMessage")] public string? CastMessage { get; set; } = default!;
+        [ViewVariables] [DataField("cooldown")] public float CoolDown { get; set; } = 1f;
+        [ViewVariables] [DataField("spellItem")] public string ItemProto { get; set; } = default!;
```

Comment:
Leave a TODO that this needs to be an EntityPrototype as no validation is being done on it (and we don't have the serializer for it yet AFAIK)